### ISSUE TITLE
wvfLoadWavefrontOpticsData broken

### ISSUE THREAD

**Issue by DavidBrainard** [isetbio/isetbio#382]
This function is broken, at least when called from the tutorial that exercises it and the examples in its source.  The tutorial is t_wvfJaekenArtal2012.

I did a little sleuthing in github. The tutorial ran OK up until a commit from Eline on 8/30,18, when the Polans data were added.

The error message is below.  My guess is that this is some simple bookkeeping bug.  I'm hoping @elinekupers will immediately be able to understand and fix the problem.

Index exceeds array bounds.

Error in wvfLoadWavefrontOpticsData (line 288)
                find(ismember(vertEccen, round(params.eccentricity(2))))]; ... Vertical

Error in t_wvfJaekenArtal2012 (line 48)
[wvf, oi] = wvfLoadWavefrontOpticsData('source', 'JaekenArtal2012', 'jIndex', zIndices, 'whichEye', whichEye, ...

**Comment by DavidBrainard** [comment]
After I merged in Jenn's most recent documentation updates, t_t_wvfJaekenArtal2012 started working (although it had a warning from wvfLoadWavefrontOpticsData whichI made go away by fixing the call to pass both horiz and vert eccentricity explicitly).

And the first of the examples in wvfLoadWavefrontOpticsData started working.

The second example remains broken and needs attention.

**Comment by elinekupers** [comment]
Thanks for letting me know! I will look into it.

**Comment by elinekupers** [comment]
It was an indexing error (classic). I pushed my changes. Both examples and the tutorial should now run without errors. 

The warning at the beginning of running the second example is caused by the single eccentricity value as an input. For the Jaeken and Artal 2012 dataset, that is ok. There is only data along the horizontal axis. For the Polans et al. 2014 data set, there are data along both horizontal and vertical axes. In the case of only one eccentricity value, the function assumes that one eccentricity value corresponds to the horizontal axis (ignoring the vertical) and spits out a warning.

I also fixed some small things like comments, printed text, and warnings.
